packages/ckb-indexer/tests/transaction_collector.test.ts

```diff
@@ -161,3 +196,43 @@ test("input cell can be found in transaction detail", async (t) => {
   getTransactionsStub.restore();
   requestBatchStub.restore();
 });
+
+test("should add inputCell to all transaction which txHash and ioType is same as query", async (t) => {
+  const searchKey: SearchKey = {
+    script: {
+      code_hash:
+        "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8",
+      hash_type: "type",
+      args: "0xdfc8ad219178a307ff117ee4e2fe760cd18410a7",
+    },
+    script_type: "lock",
+    filter: {},
+  };
+  const SearchFilter = { sizeLimit: undefined, order: undefined };
+  getTransactionsStub
+    .withArgs(searchKey, SearchFilter)
+    .returns(multipleInputCellTx);
```

Comment:
maybe the returns should wrap with a Promise?

Reply:
yes,multipleInputCellTx is a Promise, it have be wrapped at https://github.com/sighWang/lumos/blob/fix-transaction-collector-bug/packages/ckb-indexer/tests/transaction_collector_special_test_case.ts#L278

Reply:
i see